Uplift/Controllers/FilterViewController.swift

```diff
@@ -333,47 +354,112 @@ class FilterViewController: UIViewController, RangeSeekSliderDelegate {
         }
 
         // CLASS TYPE SECTION
-        classTypeDropdown.snp.remakeConstraints { make in
+        classTypeDropdownHeader.snp.remakeConstraints { make in
             make.top.equalTo(startTimeClassTypeDivider.snp.bottom)
             make.left.right.equalToSuperview()
+            make.height.equalTo(55)
+        }
+        
+        classTypeDropdown.snp.remakeConstraints { make in
+            make.top.equalTo(classTypeDropdownHeader.snp.bottom)
+            make.left.right.equalToSuperview()
 
             switch classTypeDropdownData.dropStatus! {
             case .up:
-                make.height.equalTo(55)
+                make.height.equalTo(0)
             case .half:
-                make.height.equalTo(55 + 32 + 3*32)
+                make.height.equalTo(3*32)
             case .down:
-                make.height.equalTo(55 + 32 + classTypeDropdown.numberOfRows(inSection: 0)*32)
+                make.height.equalTo(classTypeDropdown.numberOfRows(inSection: 0)*32)
             }
         }
 
+        classTypeDropdownFooter.snp.remakeConstraints { make in
+            make.top.equalTo(classTypeDropdown.snp.bottom)
+            make.left.right.equalToSuperview()
+
+            switch classTypeDropdownData.dropStatus! {
+            case .up:
+                make.height.equalTo(0)
+            case .half, .down:
+                make.height.equalTo(32)
+            }
+        }
+        
         classTypeInstructorDivider.snp.remakeConstraints { make in
             make.width.centerX.equalToSuperview()
-            make.top.equalTo(classTypeDropdown.snp.bottom)
-            make.bottom.equalTo(classTypeDropdown.snp.bottom).offset(1)
+            make.top.equalTo(classTypeDropdownFooter.snp.bottom)
+            make.bottom.equalTo(classTypeDropdownFooter.snp.bottom).offset(1)
```

Comment:
you can do `make.bottom.equalTo(classTypeDropdownFooter).offset(1)`, SnapKit will automatically assume you mean `bottom` if you don't specify it